### PR TITLE
Add heartbeat every 10ms

### PR DESCRIPTION
**Pull request by Graicc**

### CODE REVIEW

**Review by Graicc — COMMENTED** [review]
Also not tested

File: Kraken/interface.py [hunk 1]
```diff
@@ -229,6 +232,11 @@ def _readline(self, serial: serial.Serial, eol: bytes) -> bytes:
 
         return bytes(line)
 
+    def serial_listen(self, serial_conn: serial.Serial):
```
Comment (Aman-Anas) [inline Kraken/interface.py]: function header seems a bit wack

**Review by Graicc — COMMENTED** [review]
Fix naming

File: Kraken/interface.py (hunk 1 above)
Comment (Graicc) [reply Kraken/interface.py]: oueps

**Review by Aman-Anas — APPROVED** [review]

**Pull request merged**